### DISCUSSION TITLE
Sample Java script to access the DOM of the active web tab

### DISCUSSION THREAD

**Discussion by tsong697** [microsoft/MicrosoftEdge-Extensions#149] [Q&A]
Can anybody show me a sample script to access the DOM of the active page in Edge browser? where the script should be in the extension source files? and how to write data onto hard disk or push the data to some kind of server listening on some kind of communication protocols?

**Comment by ManikanthMSFT** [comment]
Hey @tsong697, here are the samples we have available right now: https://learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/samples.  Please review them and let us know if you have any further concerns.
